CONTRIBUTING.md

```diff
@@ -1,11 +1,51 @@
 # Contribution guidelines
 
-So you want to hack on Knative Eventing Kafka Broker? Yay! Please refer to
-Knative's overall
-[contribution guidelines](https://www.knative.dev/contributing/) to find out how
-you can help.
+So you want to hack on Knative Eventing Kafka Broker? Yay! Please refer to Knative's overall
+[contribution guidelines](https://www.knative.dev/contributing/) to find out how you can help.
 
-# Useful links to get started
+## Getting started
+
+1. [Create and checkout a repo fork](#checkout-your-fork)
+
+### Requirements
+
+You need to install:
+
+- [`ko`](https://github.com/google/ko) - (_required_)
+- [`docker`](https://www.docker.com/) - (_required_)
+- [`Go`](https://golang.org/) - (_required_)
+    - check
+      [go \<version\>](https://github.com/knative-sandbox/eventing-kafka-broker/blob/master/go.mod)
+      for the required Go version used in this project
+- [`Java`](https://www.java.com/en/) (we recommend an `openjdk` build) -
+  (_required_)
+    - check
+      [java.version](https://github.com/knative-sandbox/eventing-kafka-broker/blob/master/data-plane/pom.xml)
+      maven property for the required Java version used in this project
+
```

Comment:
You should add as requirements here:

* `protoc`. Specify that the version to use is here https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/contract/contract.pb.go#L4. As guide to install it, you can point this one https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os
* `protoc-gen-go` Specify that the version to use is here https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/contract/contract.pb.go#L3. To install it:

```
go install google.golang.org/protobuf/cmd/protoc-gen-go@[version]
```